deployments/registry.go

```diff
@@ -194,9 +217,13 @@ func (k Registry) apply(ctx context.Context, c *kubernetes.Cluster, ui *termui.U
 	}
 
 	// Wait for the cert manager to be present and active. It is required
+	log.Info("waiting for cert manager to be present and active")
```

Comment:
I was wondering about the missing logger in those deployments 👍

Reply:
Should be newly declared for the structure, in the `install_client` code, and as new argument of `apply`.